CONTRIBUTING.md

```diff
@@ -101,6 +101,60 @@ get going:
   your SSH key passphrase to the managing agent, so that you don't have to
   keep authenticating to GitHub. You need to do this again after every reboot.
 
+## Supplementary Developer Contribution Guidelines:
+1. Ensuring to squash similiar commits  
+To ensure that your pull request does not contain multiple similiar commits, you may need to squash your commits.
+Here's how to do it:
+
+    **Rebase Your Branch:**   
+    Rebase your feature branch onto the latest version of the target branch
+    ```
+    git checkout feature-branch
+    git fetch origin
+    git rebase origin/master
+    ```
+    **Squash Commits:**  
+    Interactive rebase to squash commits:
+    ```
+    git rebase -i HEAD~N
+    ```
+    Replace 'N' with the number of similiar commits you want to squash. In the interactive rebase interface, 
+    replace 'pick' with 'squash' for all commits except the first one. And amend the commit message if needed.
+
+    **Force Push:**  
+    After squashing your commits, force push your branch to update your pull request:
+    ```
+    git push -f 
+    ```
+
+2. DCO (Developer Certificate of Origin) needed for each commit  
+Contributors must sign off on their commits to certify that they have the right

```

Comment:
The first half of the guide mentioned the need to add a signature when committing. Is it possible to add a **for more details** link in the corresponding position in the first half to point to this new part?

![details](https://github.com/kubeedge/website/assets/48212997/405bef25-323a-452e-a6dc-c9c399e6028c)

Reply:
I have created a new supplemental developer guide file under the Developer Guide of https://kubeedge.io/docs/category/developer-guide.
And create a link in contributing document to link to DCO details.